_blog.yml

```
@@ -4992,4 +4992,11 @@
     - Claude
     - Codex
     - Gemini
-    - agents
\ No newline at end of file
+    - agents
+
+- local: deh-2025
```

```suggestion
- local: dell-enterprise-security
```

I think better for seo